content/en/core/releases/3.14.0.md

```diff
@@ -0,0 +1,143 @@
+- [#7186](https://github.com/medic/cht-core/issues/7186): Changing an offline user's password during replication will block them in loading error loop
+- [#7187](https://github.com/medic/cht-core/issues/7187): Login page redirect loop when password is changed
+- [#7242](https://github.com/medic/cht-core/issues/7242): Race condition occurs when logging in as offline user with slow connection
+- [#7262](https://github.com/medic/cht-core/issues/7262): Can't navigate to "new district" form when contact "person" edit is open.
+- [#7280](https://github.com/medic/cht-core/issues/7280): Server Side Purge does not complete to the end
+- [#7305](https://github.com/medic/cht-core/issues/7305): Blank modal when navigating to app instance with missing cookie
+- [#7307](https://github.com/medic/cht-core/issues/7307): Login page is not cached correctly
+- [#7320](https://github.com/medic/cht-core/issues/7320): Admin Upgrade Releases section empty when there are no betas
+- [#7343](https://github.com/medic/cht-core/issues/7343): PWA on mobile logs out when browser is quit
+- [#7346](https://github.com/medic/cht-core/issues/7346): Show error in targets when user is not associated with the configured place
+- [#7353](https://github.com/medic/cht-core/issues/7353): Partner logos don't show on the about page
+- [#7383](https://github.com/medic/cht-core/issues/7383): Webapp relative dates are not localized after Angular 12 upgrade
+- [#7419](https://github.com/medic/cht-core/issues/7419): Stopping and resuming initial replication could start local purging
+- [#7438](https://github.com/medic/cht-core/issues/7438): Users can view reports in contacts detail page despite lacking permissions
+- [#7463](https://github.com/medic/cht-core/issues/7463): Incorrect place filter label when selecting only one place
+- [#7485](https://github.com/medic/cht-core/issues/7485): Error while submitting form with certain db-doc repeat
+
+### Technical improvements
+
+- [#6420](https://github.com/medic/cht-core/issues/6420): Move to ESLint 7
+- [#7061](https://github.com/medic/cht-core/issues/7061): CHT hosting improvements for App Developers
+- [#7083](https://github.com/medic/cht-core/issues/7083): Upgrade webapp to latest Angular
+- [#7165](https://github.com/medic/cht-core/issues/7165): Release Notes for v3.8 have link to private repository (broken link for external users)
+- [#7349](https://github.com/medic/cht-core/issues/7349): Update dependencies for 3.14
+- [#7361](https://github.com/medic/cht-core/issues/7361): Upgrade google-libphonenumber
+- [#7379](https://github.com/medic/cht-core/issues/7379): Update COVID-19 RDT reference app
+- [#7413](https://github.com/medic/cht-core/issues/7413): Remove non-code files from repo
+- [#7445](https://github.com/medic/cht-core/issues/7445): Fix reference to `install.md` that does not exist any more
+- [#7453](https://github.com/medic/cht-core/pull/7453): Extend `add-local-ip-certs-to-docker.sh` to install expired and self signed certs
```

Comment:
I think this one should be removed from the milestone and release notes - it's in the repo but not part of the app per se.